spec/poros/breweries_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+RSpec.describe "Brewery Poros" do
```

Comment:
This works, but I believe convention is to user `RSpec.describe Class` when we are testing a class.